airflow/providers/google/cloud/sensors/gcs.py

```diff
@@ -390,6 +391,7 @@ class GCSUploadSessionCompleteSensor(BaseSensorOperator):
         If set as a sequence, the identities from the list must grant
         Service Account Token Creator IAM role to the directly preceding identity, with first
         account from the list granting this role to the originating account (templated).
+    :param deferrable: bool = False
```

Comment:
would be nice to describe this param instead of `bool = False`

Reply:
done